README.md

```diff
@@ -50,54 +50,31 @@ Follow these steps to create the required services and run the notebook locally.
 
 1. [Setup HDP platform](#1-set-up-hdp-platform)
 2. [Setup HDP Search](#2-set-up-hdp-search)
-3. [Setup DSX Local](#3-set-up-dsx-local)
-4. [Setup a Livy gateway](#4-set-up-a-livy-gateway)
-5. [Download the Solr Spark connector](#5-download-the-solr-spark-connector)
-6. [Clone the repo](#6-clone-the-repo)
-7. [Download and move the data to HDFS](#7-download-and-move-the-data-to-hdfs)
-8. [Setup python plugins](#8-setup-python-plugins)
-9. [Launch the notebook](#9-launch-the-notebook)
-10. [Run the notebook](#10-run-the-notebook)
+3. [Setup DSX Desktop](#3-set-up-dsx-desktop)
+4. [Download the Solr Spark connector](#5-download-the-solr-spark-connector)
+5. [Clone the repo](#6-clone-the-repo)
+6. [Download and move the data to HDFS](#7-download-and-move-the-data-to-hdfs)
+7. [Setup python plugins](#8-setup-python-plugins)
+8. [Launch the notebook](#9-launch-the-notebook)
+9. [Run the notebook](#10-run-the-notebook)
 
 ### 1. Setup HDP platform
 
-Due to the various deployment methods of HDP we will cover the method we used when developing the code pattern, which was to use a Dockerized HDP Sandbox. This can be accomplished by performing the following steps:
+This code pattern was tested ony HDP v2.6.4, follow [this link](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/bk_command-line-installation/content/ch_getting_ready_chapter.html) to install and setup HDP cluster. 
```

Comment:
typeo. How about this ?

Please install HDP v2.6.4 by following instructions at [install_hdp](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/bk_command-line-installation/content/ch_getting_ready_chapter.html).

Reply:
ok